tools/applicationinsights-web-snippet/src/type.ts

```diff
@@ -1,9 +1,18 @@
 import { IConfiguration } from "@microsoft/applicationinsights-web";
 
-// in the future, we can add more fields here
 export interface SdkLoaderConfig {
     instrumentationKey?: string;
     connectionString?: string;
+    src?: string;
+    name?: string;
+    ld?: number;
+    useXhr?: boolean;
+    crossOrigin?: string;
+    onInit?: any;
```

Comment:
onInit: should be defined as a function -- although I'm not sure that the JSON.stringify() is going to do what we want